src/scripts/main.js

```diff
@@ -1,7 +1,30 @@
 'use strict';
 
 const pushNotification = (posTop, posRight, title, description, type) => {
-  // write code here
+  const notification = document.createElement('div');
+
+  notification.classList.add('notification', type);
+
+  const titleElement = document.createElement('h2');
+
+  titleElement.classList.add('title');
+  titleElement.textContent = title;
+  notification.appendChild(titleElement);
+
+  const descriptionElement = document.createElement('p');
+
+  descriptionElement.classList.add('description');
+  descriptionElement.textContent = description;
+  notification.appendChild(descriptionElement);
+
+  notification.style.top = `${posTop}px`;
+  notification.style.right = `${posRight}px`;
+
+  document.body.appendChild(notification);
+
+  setTimeout(() => {
+    notification.style.visibility = 'hidden';
+  }, 2000);
```

Comment:
Here, you're hiding the notification after 2 seconds, but the DOM element is still present. Over time, if a lot of notifications are created, this could lead to performance issues. Consider removing the notification from the DOM instead of just hiding it.